src/utils.js

```diff
@@ -0,0 +1,33 @@
+import { resourceEndpoints }  from './routes/base';
+
+/*
+ * Module with utility shared methods
```

Comment:
It seems that we are only using this on `sensorthings`. Do we need it to be an independent module?